docs/src/apimanual.md

```diff
@@ -178,9 +178,13 @@ By convention, solvers are not expected to support nonzero constant terms in the
 | Mathematical Constraint                                       | MOI Function                 | MOI Set                            |
 |---------------------------------------------------------------|------------------------------|------------------------------------|
 | ``\lVert Ax + b\rVert_2 \le c^Tx + d``                        | `VectorAffineFunction`       | `SecondOrderCone`                  |
+| ``\lVert x[2:end] \rVert_2 \le x[1]``                         | `ScalarVariablewiseFunction` | `SecondOrderCone`                  |
```

Comment:
I'd rather use latex notation than `[2:end]` here. I think it's fine to say ``y \ge \lVert x \rVert` and let users refer to the documentation for the variable ordering. In the line above we don't make a point of the order of the affine functions.